modules/samples/artifacts/ReceiveHTTPInJsonFormatWithCustomMapping/ReceiveHTTPInJsonFormatWithCustomMapping

```diff
@@ -0,0 +1,72 @@
+
+@App:description('Receive events via HTTP transport in JSON format with custom mapping and view the output on the console')
+/*
+Purpose:
+    This application demonstrates how to configure WSO2 Stream Processor to receive sweet production events via HTTP transport in JSON format using custom mapping and log the  events in LowProducitonAlertStream to the  output  console.
+
+Prerequisites:
+    1) Save this sample
+
+Executing the Sample:
+    1) Start the Siddhi application by clicking on 'Run'.
+    2) If the Siddhi application starts successfully, the following messages would be shown on the console.
+        * Source Listener has created for url http://localhost:8006/productionStream
+        * ReceiveHTTPInJsonFormatWithDefaultMapping.siddhi - Started Successfully!
+
+Testing the Sample:
+    Option1: Publish events with http sample client:
+    Navigate to {WSO2SPHome}/samples/sample-clients/http-client and run "ant" command with arguments.
+        ant -DcustomMapping=true
+    If you want to publish custom number of events, you need to run "ant" command as follows
+        ant -DcustomMapping=true -DnoOfEventsToSend=5
+
+    Option2: Publish events with curl command:
+    Publish few events in json format to the http endpoint as follows (The values for name and amount attributes can be changed as desired)
+
+        curl -X POST -d "{\"item\": {\"id\":\"Sugar\",\"amount\": 300.5}}"  http://localhost:8006/productionStream --header "Content-Type:application/json"
+
+
+    Option3: Publish events with Postman:
+        a) Install 'Postman' application from Chrome web store.
+        b) Launch the application.
+        c) Make a 'Post' request to 'http://localhost:8006/productionStream' endpoint. Set the Content-Type to 'application/xml' and set the request body in json format as follows,
+
+             {
+                "item": {
+                    "id": "sugar",
+                    "amount": 20.0,
+                }
+            }
+
+Notes:
+    If you edit this application while it's running, stop the application -> Save -> Start.
+    If the message "Source Listener has created for url http://localhost:8006/productionStream" does not appear,it could be due to port 8006, defined in the Siddhi application is already being used by a different program. To resolve this issue, please do the following,
+        * Stop this Siddhi application (Click 'Run' on menu bar -> 'Stop')
+        * Change the port 8006 to an unused port, in this Siddhi application's source configuration.
+        * Start the application and check whether the specified messages appear on the console
+
+Viewing the Results:
+    See the output. Following message would be shown on the console.
+    ReceiveHTTPInXMLFormatWithDefaultMapping : LowProducitonAlertStream : Event{timestamp=1511938781887, data=[sugar, 300.0], isExpired=false}
+
+
+    Note:
+    Stop this Siddhi application, once you are done with the execution
+
+*/
+@Source(type = 'http',
```

Comment:
For conformity shall we use simple letters for annotations? i.e source instead of Source